# Goal
geometry_msgs/PoseStamped goal_point
std_msgs/Float32MultiArray scooping_trajectory
float32 angle
string function_name

---

# Result
bool success

---

string status